src/styles/blocks/form-field.scss

```diff
@@ -0,0 +1,35 @@
+.form-field {
+  box-sizing: border-box;
+  display: block;
+  height: 48px;
+  width: 100%;
+  margin: 0 0 16px;
+  padding: 14px 16px;
+  font-size: 14px;
+  line-height: 20px;
+  color: $colorBlack;
+  background-color: $formBackgroundColor;
+  border-radius: 40px;
```

Comment:
Consider using a variable for the border radius to maintain consistency and make it easier to update in the future. For example: $borderRadiusLarge: 40px;